ocm_python_wrapper/cluster.py

```diff
@@ -278,23 +290,30 @@ def validate_addon_parameters(self, parameters):
                 f"{self.addon_name} missing some required parameters {missing_parameter}"
             )
 
-    def install_addon(self, parameters=None, wait=True, wait_timeout=TIMEOUT_30MIN):
+    def install_addon(
+        self,
+        parameters=None,
+        wait=True,
+        wait_timeout=TIMEOUT_30MIN,
+        rosa_cli=False,
```

Comment:
```suggestion
        rosa=False,
```

`run_action` passes `rosa` param to `install_addon` kwargs as shown [here](https://github.com/RedHatQE/ocp-addons-operators-cli/blob/85a0cbe9c9f054bed19b0f80ac51aa8632a5ed21/app/addon_cmds/commands.py#L16)